.github/workflows/Specs.yml

```diff
@@ -0,0 +1,48 @@
+name: Specs
+
+jobs:
+  specs:
+    strategy:
+      matrix:
+        os: [ubuntu-16.04]
+        ruby: [2.3.3, 2.4, 2.5, 2.6, 2.7]
+        include:
+          - os: macos-10.15
+            ruby: system
+
+    name: Ruby ${{ matrix.ruby }}
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Checkout git
+        uses: actions/checkout@v1
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        if: ${{ matrix.ruby != 'system' }}
+        with:
+          ruby-version: ${{ matrix.ruby }}
+
+      - name: Update git submodules
+        run: git submodule update --init
+
+      - name: Setup test_repo
+        run: ./.github/scripts/setup_test_repo.sh
+
+      - name: Run bundle install
+        run: |
+          gem install bundler -v "~> 1.17"
+          bundle install --jobs 4 --retry 3 --without debugging documentation
+
+      - name: Run Specs + Rubocop
+        run: bundle exec rake spec
+
+on:
+  push:
+    branches:
```

Comment:
Is there a reason to not run on all branches? (Getting a ci run before opening the PR itself can be nice sometimes)

Reply:
@endocrimes It used to be the case that this caused a weird duplication in test runs that would show up in checks. I'll try and see if it's still the case.

Reply:
@endocrimes Yup, it did that. Reverting 😭